tests/test_aips_models.py

```diff
@@ -3,7 +3,7 @@
 import unittest
 
 from aips import models
-from tests.test_amclient import TmpDir
+from tests.tests_helpers import TmpDir
```

Comment:
Maybe `tests.helpers` is a better name? Or `tests.tmpdir`?

It reminds me that at some point we should just move over the [`tmp_path`](https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture) pytest fixture.